.github/workflows/sdk.yml

```diff
@@ -0,0 +1,37 @@
+name: sdk
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  testing-sdk:
+    name: Test Pipcook SDK
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ['12']
+        os: [ubuntu-latest, macOS-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Using Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+    - name: Building Pipcook
+      run: |
+        npm install
+        npm run build
+    - name: pipcook init
+      env:
+        NODE_ENV: local
+      run: |
+        ./packages/cli/dist/bin/pipcook init
+        ./packages/cli/dist/bin/pipcook daemon start
+    - name: pipcook sdk test
+      run: |
+        node ./packages/sdk/test/job.js
+        node ./packages/sdk/test/pipeline.js
```

Comment:
We need to run tests with https://github.com/alibaba/pipcook/blob/master/run_tests.ts